background/redirects.rst

```diff
@@ -0,0 +1,88 @@
+.. _redirects:
+
+Redirects in Divio Cloud projects
+=================================
+
+You will find that various redirects at work on URLs in Divio Cloud projects.
+
+They fall into different categories:
+
+* protocol redirects (e.g. ``http://example.com`` to ``https://example.com``)
+* domain name redirects (e.g. ``https://www.example.com`` to ``https://example.com``)
```

Comment:
Let's take redirecting from ``https://example.com`` to ``https://www.example.com`` as an example here instead of the other way around. Using the base domain (``example.com``) has its downsides as you can't set a ``CNAME`` DNS record to point to our load-balancer or CloudFlare. So we should encourage the ``www.example.com`` version in examples.